sdks/python/apache_beam/runners/runner.py

```diff
@@ -84,6 +85,10 @@ def create_runner(runner_name):
         raise ImportError(
             'Google Cloud Dataflow runner not available, '
             'please install apache_beam[gcp]')
+      elif 'interactive' in runner_name.lower():
+        raise ImportError(
```

Comment:
Have you tested this to be working as expected?

Reply:
Yes, tested locally by moving the `_ALL_KNOWN_RUNNERS` for `InteractiveRunner` to a non-existing path and when creating the runner, it raises the ImportError.